README.md

```diff
@@ -26,10 +27,10 @@ Depend on Beeju using:
 ```
 
 ### JUnit4
-For JUnit4, ensure you have the JUnit4 dependency in your POM.
+For JUnit4, ensure you have the JUnit4 dependency in your POM, as Beeju no longer supplies it as a transitive dependency.
 
-### JUnit5
-For JUnit5, support is available to enable you to migrate your tests that currently use Beeju rules. To use JUnit5, ensure you have:
+### JUnit5 Rule Migration
```

Comment:
Maybe redo this and have JUnit5 documented first with just examples of the extensions and then a separate section afterwards for JUnit4 which just has the rules. I know this has more repeated code but for readers they can just look at one section and don't have to think about whether they're doing a rule or an extension.